serversideaction.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+from vi.config import conf
+from vi.html5 import Button, window
+from vi.safeeval import SafeEval
+import logging
+
+
+class ServerSideActionWdg(Button):
```

Comment:
The purpose of this entire functionality is unclear. No examples, no documentation, no hint at all. This will be removed when there is no usable and helpful docstring at least.

Reply:
@tsteinruecken